### PR TITLE
Feature - Enable Logic Apps by config

### LINKED ISSUE #7: Provide script to enable an Azure Logic Apps based on configuration-file
Provide script to enable an Azure Logic App

### PR DESCRIPTION
**Pull request by mbraekman**
Add the counter-part to the Disable-AzLogicAppsByConfig-command, allowing the use of the same configuration file to enable Azure Logic Apps in a given order.

Closes #7

### CODE REVIEW

File: src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1 [hunk 1]
```diff
@@ -26,6 +38,18 @@ Describe "Arcus" {
                 Assert-MockCalled Get-AzLogicAppRunHistory -Times 0
                 Assert-MockCalled Get-AzLogicApp -Times 0
             }
+            It "Doesn't enable anything when both checkType & stopType is 'None'" {
+                # Arrange
+                Mock Get-AzLogicAppRunHistory {}
```
Comment (stijnmoreels) [inline src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1:43]: Some backdoor parameter assertion can be added here so we can verify that the correct values are send to these mocked functions. (Like `State = Enabled`, `ResourceGrouop = 'ignored-resource-group'`)

File: docs/preview/features/powershell/azure-logic-apps.md [hunk 2]
```diff
@@ -93,29 +94,155 @@ PS> Disable-AzLogicAppsFromConfig -DeployFilename "./deploy-orderControl" -Resou
...
+
+Typically done as the last task of the release pipeline, right after the deployment of the Logic Apps, as this will enable all specified Logic Apps in a specific order. 
+The Azure Logic Apps to be enabled and the order in which this will be done, will be defined in the provided configuration file.
+The order of the Azure Logic Apps in the configuration file (top to bottom) defines the order in which they will be enabled by the script. The counterpart of this script used to disable the Azure Logic Apps, will take the reversed order as specified (bottom to top) in the file.
+
+| Parameter         | Mandatory | Description                                                                                                         |
+| ----------------- | --------- | ------------------------------------------------------------------------------------------------------------------- |
+| ResourceGroupName | yes       | The resource group containing the Azure Logic Apps.                                                                 |
+| DeployFileName    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file. |
```
Comment (stijnmoreels) [inline docs/preview/features/powershell/azure-logic-apps.md:146]:
```suggestion
| Parameter         | Mandatory | Description                                                                                                           |
| ----------------- | --------- | --------------------------------------------------------------------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the Azure Logic Apps.                                                                   |
| `DeployFileName`    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file.   |
```

**Review by stijnmoreels — APPROVED** [review]

File: src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1 (hunk 1 above)
Comment (mbraekman) [reply src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1:43]: Good remark, let's come back to this and perhaps review all tests to include these type of assertions from a different issue?

Comment (stijnmoreels) [reply src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1:43]: Hmm, as far as I know we've already added these backdoor assertions in all tests. But ok.

File: docs/preview/features/powershell/azure-logic-apps.md (hunk 2 above)
Comment (stijnmoreels) [reply docs/preview/features/powershell/azure-logic-apps.md:146]: Shame this suggestion isn't part of the merge; as it makes sure that we have consistent docs.

**Pull request merged**